indico/htdocs/sass/partials/_icons.scss

```diff
@@ -96,441 +96,591 @@ i[class^="icon-"]:before {
 .icon-agreement:before {
     content: "\e605";
 }
+
```

Comment:
i wouldn't add all these empty lines... it's a bit noisy considering it's just generated code anyway